apps/github-stack/src/github-ineraction/domain/scheduler/github-scheduler.ts

```diff
@@ -51,8 +51,8 @@ export class GitHubScheduler {
     const repositories = await this.gitRepository.find({ relations: ["user"] });
     for (const repo of repositories) {
       const release = await this.getLatestReliase(repo);
-      if (repo.latestRelease != release) {
-        repo.latestRelease = release;
+      if (repo.release != release) {
+        repo.release = release;
```

Comment:
Hm, you didn't change this so it's still the same issue as when we talked (storing some object in arrray of Relese entities)